controller/model/base_handler.go

```diff
@@ -186,6 +186,65 @@ func (handler *baseHandler) patchEntity(modelEntity boltEntitySource, checker bo
 	return handler.updateGeneral(modelEntity, checker, true)
 }
 
+func (handler *baseHandler) patchEntityBatch(modelEntity boltEntitySource, checker boltz.FieldChecker) error {
+	return handler.updateGeneralBatch(modelEntity, checker, true)
+}
+
+func (handler *baseHandler) updateGeneralBatch(modelEntity boltEntitySource, checker boltz.FieldChecker, patch bool) error {
```

Comment:
This is a stand-in copy of `updateGeneral()` with log output commented out.

`Batch`  writes to bolt can be rerun and thus produce extra log output. This implementation should evolve as we understand our needs for `Batch` writes.

Reply:
I was thinking about this again, and I feel like the log messages are ok, since they are reporting errors. I'm not clear on batch behavior, but I suspect that if your function returns an error, it should not be retried. Generally if it fails once, it will keep failing

Reply:
Uncommented, added "batch:" prefix to differentiate the log messages.